### PR TITLE
'For Virtual Space Owners' Category Update

### PR DESCRIPTION
**Pull request by philburrrt**
Issue: #6 

Notes:
- Reviewed all copy, manually went through all guides
- Gave WebXR Integrations its own category so readers don't have to scroll to see supported platforms
- Added links to repositories, banner attribute documentation and references to source code
- Added screenshots to platform integrations where appropriate
- Changed all README.md to table of contents for corresponding category. Parsed copy from README's to create a better and more clear information flow

### CODE REVIEW

File: guides/for-creators/create-auction.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+From the [Space](https://app.zesty.market/spaces) page, select the Space you want to create an auction with.
+
+Next, click on **Create Auction** under the Auctions section. The Auction calendar will pop-up. Afterwards:
+
+1. Choose a date or multiple dates for the auction slot.
+2. Set the USDC cost for the created auction slots
+3. Click **Confirm**
+
+*Note: You can set up multiple auction timeslots in a single trasnaciton*
```
Comment (limbofeather) [inline guides/for-creators/create-auction.md:11]: s/trasnaciton/transaction

File: guides/for-creators/platform-integrations/cryptovoxels-integration.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+**At the end of the script**, copy this line to call the loadBanner function (modify according to your own space information):
+
+```
+loadBanner('0', '0x0000000000000000000000000000000000000000', 'polygon', 'tall', 'standard');
+```
+
+Replace `0` and  `0x0000000000000000000000000000000000000000` with your space number and wallet address
+
+To opt into enabling a beacon on your space, copy and paste:
```
Comment (limbofeather) [inline guides/for-creators/platform-integrations/cryptovoxels-integration.md:58]: Do you think you could write a short blurb about what a beacon is?

File: SUMMARY.md [hunk 3]
```diff
@@ -9,11 +9,19 @@
...
-  * [Unity Integration](guides/for-creators/unity-integration.md)
-  * [Cryptovoxels Integration](guides/for-creators/cryptovoxels-integration.md)
-  * [Decentraland Integration](guides/for-creators/decentraland-integration.md)
-  * [Web & Twitch Integration (OBS)](guides/for-creators/web-and-twitch-integration-obs.md)
+  * [Create a Space](guides/for-creators/create-space.md)
+  * [Platform Integrations](guides/for-creators/platform-integrations/README.md)
+    * [Unity Integration](guides/for-creators/platform-integrations/unity-integration.md)
+    * [Cryptovoxels Integration](guides/for-creators/platform-integrations/cryptovoxels-integration.md)
+    * [Decentraland Integration](guides/for-creators/platform-integrations/decentraland-integration.md)
+  * [WebXR Integrations](guides/for-creators/webxr-integrations/README.md)
```
Comment (limbofeather) [inline SUMMARY.md:17]: This is a nice way to separate, thanks

File: guides/for-creators/README.md [hunk 4]
```diff
@@ -4,127 +4,12 @@ description: >-
...
-And you're done!
-
-## 4. Getting paid
-
-After the slot has been purchased and there is sufficient proof that you had shown for the agreed period of time, you'll be able to withdraw the amount used to purchase your Zesty NFT.
-
-Please refer to the video below for the step-by-step guide on how to collect your funds once the advertisement service has been completed
-
-{% embed url="https://www.youtube.com/watch?v=UKwtTbvmlL4" %}
+1. [Creating a Space](./create-space.md)
```
Comment (limbofeather) [inline guides/for-creators/README.md:11]: For some reason all of these are number 1?

**Comment by limbofeather** [comment]
I'll leave integration instruction reviews for @msub2 to check out, but for the time being everything else looks good to me.

File: guides/for-creators/webxr-integrations/r3f.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+**NPM Project** - install it like so:
+
+```
+npm install `@zestymarket/r3f-sdk'
+```
+
+Once installed, import the ZestyBanner component:
+
+```
+import * as Zesty from '@zestymarket/threejs-sdk';
```
Comment (msub2) [inline guides/for-creators/webxr-integrations/r3f.md:24]: This should be @zestymarket/r3f-sdk.

File: guides/for-creators/webxr-integrations/r3f.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+**Source**
+
+```
+<ZestyBanner
+   space={'YOUR_SPACE_ID'}
+   creator={'YOUR_CREATOR_ID'}
+   format={'YOUR_SPACE_FORMAT'}
+   style={'YOUR_DESIRED_BANNER_STYLE'}
+   position={[X, Y, Z]} />
```
Comment (msub2) [inline guides/for-creators/webxr-integrations/r3f.md:106]: This example should have the beacon field added too, ie `beacon={true}`

**Comment by msub2** [comment]
Couple nits but otherwise looks good to me

File: guides/for-creators/README.md (hunk 4 above)
Comment (philburrrt) [reply guides/for-creators/README.md:11]: Markdown syntax turns 1's into properly numbered list. Sorry, should have linked the frontend here https://philbertburrow.gitbook.io/zesty-docs/guides/for-creators

**Comment by philburrrt** [comment]
@msub2 @limbofeather everything should be fixed. pushed in [revision 2](https://github.com/zestymarket/docs/pull/7/commits/1b95e89c05c6efb76caf470c77a5ece64b6b0eec)

**Comment by limbofeather** [comment]
If things look good to @msub2, LGTM.

**Review by limbofeather — APPROVED** [review]
🚢

**Review by msub2 — APPROVED** [review]
LGTM

**Pull request merged**